packages/vapor/scss/components/navigation.scss

```diff
@@ -121,16 +121,16 @@
                             height: $navigation-menu-section-height;
                             margin-right: $navigation-right-margin;
                             padding-left: $navigation-horizontal-space;
-                            color: var(--navigation-text-color);
+                            color: var(--navy-blue-80);
                             font-size: var(--default-font-size);
                             letter-spacing: 0.05em;
 
                             &.state-locked {
-                                color: var(--navigation-locked-color);
+                                color: $navigation-locked-color;
```

Comment:
I'm curious. I think it's the only color that cannot be override. Is it because it's 'locked'? Or simply we know that any navigation bar should have the same color?

Reply:
I'm not sure to understand the question, why it cannot be overridden? We did change this color from medium-grey to grey-60 https://github.com/coveo/react-vapor/pull/1893/files#diff-b533deb43c9f33669ab5f73fa10eba4f2c7035a6595dd621d0d3c9f5904693a6R243 PO did this part but I think it's based on the mockup.

Reply:
Oh you're right, I should I have check the content of this variable. My b :sweat_smile: